interactions/api/gateway.py

```diff
@@ -229,24 +230,27 @@ def handle(self, event: str, data: dict) -> None:
             path += ".models" if event == "INTERACTION_CREATE" else ".api.models"
 
             if event != "INTERACTION_CREATE":
-                obj: object = getattr(
-                    __import__(path),
-                    name.split("_")[0].capitalize(),
-                )
-
-                self.dispatch.dispatch(f"on_{name}", obj(**data))
+                try:
+                    obj: object = getattr(
+                        __import__(path),
+                        name.split("_")[0].capitalize(),
+                    )
+                except AttributeError:
+                    pass  # TODO: Actually don't dynamic this because of payloads.
```

Comment:
Howcome? The goal is that if we need to create aliasing names of classes, we can just subclass the original "base" class. That's why `TextChannel`, `DMChannel` and etc. exist, purely to be aliased as derivations of the original `Channel` class.

Reply:
This is implemented not because of derivation, this is because of 
_breaking events that crash the bot cough reactions and voice updates cough_

Granted, this is more of a development/initial test "hotfix" and my thought process is that it won't be here on production and actually have them point correctly.

Reply:
I'm inclined in wholehearted honesty to remove this part of the commit for the time being because attribute errors are still important for us to catch here for the time being. We can always revise the dispatching method. I do believe it should still be dynamic but change the way we're reading for importing objects.